src/Utilities/Microphysics/Microphysics.jl

```diff
@@ -0,0 +1,115 @@
+"""
+    Microphysics
+
+Microphysics parameterization functions (for example condensation,
+evaporation, autoconversion rates for cloud water and rain water)
+TODO: Snow, ice, and 2-moment formulation
+"""
+module Microphysics
+
+using CLIMA.MoistThermodynamics
+using CLIMA.PlanetParameters
+
+using Printf
+
+# rates of conversion between microphysics categories
+export qv2qli, ql2qr, qr2qv
+
+# rain fall speed
+export terminal_velocity
+
+# conversion from specific humidity to mixing ratio (TODO - temporary)
+export q2r
+
+"""
+    qv2qli(PhasePartition, T, ρ[, timescale=1])
+
+Return the ql and qi source terms due to condesation/evaporation (for ql)
+and sublimation/resublimation (for qi)
+The source terms are obtained assuming a relaxation to equilibrum with a
+constant timescale. The timescale is an optional parameter.
+"""
+function qv2qli(q_sa::PhasePartition, q::PhasePartition,
+                timescale::DT=DT(1)) where {DT<:Real}
+
+    dqldt = (q_sa.liq - q.liq) / timescale
```

Comment:
``dq_liq_dt`` etc. would be easier to read.

Reply:
Or better yet, and what other models use, ``d_dt_q_liq``. But because this is not entirely accurate with nonzero mass sources from precipitation, something like ``source_q_liq`` may be even better. It would be good for us to come up with a consistent naming convention for sources like this.

Reply:
Yes - lets set a convention for all the microphysics species. And I'll set the names accordingly.

Reply:
Yes, and it's not just for microphysics. This will come up in many places, for many other sources (gravity waves, radiation etc.)